buildserver/proxy_test.go

```diff
@@ -42,8 +44,9 @@ func TestProxy(t *testing.T) {
 		depFS             map[string]map[string]string // dep clone URL -> map VFS
 	}{
 		"go basic": {
-			rootURI: "git://test/pkg?deadbeefdeadbeefdeadbeefdeadbeefdeadbeef",
-			mode:    "go",
+			rootURI:         "file:///",
+			originalRootURI: "git://test/pkg?deadbeefdeadbeefdeadbeefdeadbeefdeadbeef",
```

Comment:
given we are supporting both use cases while we roll out, can't you adjust each test case to run twice. Once with the old rootURI, and one with the new way of specifying?

Reply:
✅ [Test both with and without OriginalRootURI](https://github.com/sourcegraph/go-langserver/pull/373/commits/86afb912a487cc077cfbbb55bdca5667914e37b9)